webapp/src/pages/PlayPage/PlayPage.tsx

```diff
@@ -58,9 +108,46 @@ export default function PlayPage() {
     });
   }
 
+  if (!authUser) {
+    return <LandingPage />;
+  }
+
   return (
-    <SearchContextProvider fetchFn={mockFetchFn} maxEntries={ENTRIES_LIMIT}>
-      <MainTabTemplate dataMapper={mockedMapper} buttons={buttons} />
-    </SearchContextProvider>
+    <div className="play-page">
```

Comment:
There might be some duplicated code, but I couldn't make it work with the templates because they use HTTP calls.